pkg/kubelet/cm/container_manager_windows.go

```diff
@@ -84,13 +95,32 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 	if err != nil {
 		return nil, err
 	}
+	// Correct NUMA information is currently missing from cadvisor's
+	// MachineInfo struct, so we use the CPUManager's internal logic for
+	// gathering NUMANodeInfo to pass to components that care about it.
+	numaNodeInfo, err := cputopology.GetNUMANodeInfo()
```

Comment:
This will fail always for Windows: https://github.com/kubernetes/kubernetes/blob/be1574b741cc926a77ebde551b74cdabb6a0be7f/pkg/kubelet/cm/cpumanager/topology/topology.go#L296

Will it make sense to directly pass `nil` to `NewManagerImpl` down below?